qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -5,6 +5,7 @@
     gagent_name = "org.qemu.guest_agent.0"
     gagent_install_cmd = "yum install -y qemu-guest-agent"
     gagent_start_cmd = "service qemu-guest-agent start"
+    gagent_start_cmd_hook = "qemu-ga -m virtio-serial -p /dev/virtio-ports/org.qemu.guest_agent.0 -F -d"
```

Comment:
"qemu-ga -F -d" is ok.

Reply:
Could you explain why don't need -m and -p here?

Reply:
@YanhuiMa I would suggest replace `org.qemu.guest_agent.0` with `$gagent_name`, and in the test case in Polarion there is no `-d`, could you help explain why add it here?

Reply:
It's process running in current terminal, -d is to run it in background.